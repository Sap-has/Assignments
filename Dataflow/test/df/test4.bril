@main {
  i: int = const 0;
  sum: int = const 0;
.loop:
  max: int = const 5;
  cond: bool = lt i max;
  br cond .body .end;
.body:
  sum: int = add sum i;
  one: int = const 1;
  i: int = add i one;
  jmp .loop;
.end:
  print sum;
}