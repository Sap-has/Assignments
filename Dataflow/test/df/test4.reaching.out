b1:
  in:  âˆ…
  out: i.b1.0, sum.b1.1
loop:
  in:  cond.loop.1, i.b1.0, i.body.2, max.loop.0, one.body.1, sum.b1.1, sum.body.0
  out: cond.loop.1, i.b1.0, i.body.2, max.loop.0, one.body.1, sum.b1.1, sum.body.0
body:
  in:  cond.loop.1, i.b1.0, i.body.2, max.loop.0, one.body.1, sum.b1.1, sum.body.0
  out: cond.loop.1, i.body.2, max.loop.0, one.body.1, sum.body.0
end:
  in:  cond.loop.1, i.b1.0, i.body.2, max.loop.0, one.body.1, sum.b1.1, sum.body.0
  out: cond.loop.1, i.b1.0, i.body.2, max.loop.0, one.body.1, sum.b1.1, sum.body.0
